.github/workflows/validate-xdg-metadata.yml

```diff
@@ -1,32 +1,37 @@
-name: Validate AppStream
+name: Validate XDG metadata
 
 on:
   push:
     paths:
-    - '.github/workflows/validate-appstream.yml'
+    - '.github/workflows/validate-xdg-metadata.yml'
     - 'org.musicbrainz.Picard.appdata.xml.in'
+    - 'org.musicbrainz.Picard.desktop.in'
     - 'po/appstream/*.po'
     - 'NEWS.md'
     - 'setup.py'
   pull_request:
 
 jobs:
-  validate-appstream:
+  validate-xdg-metadata:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
         python-version: '3.11'
-    - name: Install appstream-util
+    - name: Install utils
       run: |
         sudo apt-get update
-        sudo apt-get install appstream-util gettext
+        sudo apt-get install appstream-util desktop-file-utils gettext
```

Comment:
Should we use https://github.com/awalsh128/cache-apt-pkgs-action ?

Reply:
We could try. But I'd expect no real difference here, as both packages installed here are really tiny. Setting up the cache likely makes no difference. In my experience you mostly gain from caches for package installation if installing the packages require large amount of processing time outside the package. Hence I'd rather not introduce another third-party dependency in the actions if we can avoid it.

Reply:
Yup, not depending on third-party actions is better especially if there's no gain.